controllers/discovery/initDiscoveryUserData.js

```diff
@@ -69,11 +75,21 @@ const InitDiscoveryUserData = async (req, res) => {
     });
     let suggestedUsers = usersWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
+    totalData = totalData?.[0]?.total_data || 0;
+
     return res.status(200).json({
       success: true,
       message: `Fetch discovery data success`,
       suggestedUsers,
-      nextPage: page + 1
+      nextPage: page + 1,
+      total_page: totalData > 0 ? Math.ceil(totalData / limit) : 0
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation of `total_page` is done directly in the response object. While this isn't necessarily an issue, it could improve readability and maintainability to move this calculation to its own line before the response object. This way, you can easily add checks or modify the calculation if needed in the future.


<!-- This is an auto-generated comment by OSS CodeRabbit -->